### ISSUE TITLE
Is there a way to load the image without the <a/> link element?

### ISSUE THREAD

**Issue by zamirkhan** [NewOrbit/load-image-react#8]
As I understand it, `blueimp-load-image` simply renders an image on a canvas, but does not include a clickable link. This package is a way of using that package in React, but renders the image inside a clickable link to the image. Is that correct? Is there a reason for the two being different in this aspect? Thanks in advance.

**Comment by ksstott** [comment]
You're understanding is correct, this package wraps the functionality and extends it to render the image as a link. It would have been implemented like this because that would have been our requirement at the time we created this package.

It sounds sensible for this package to allow you to render an image without the link but would be a breaking change. Thank you for the suggestion; we'll consider this for a future release. We'll update as discussions take place.

**Comment by zamirkhan** [comment]
Thanks for the explanation. It's not a showstopper for me; I think some users click on it simply because they expect it to do something (because the cursor changes). 

Could it be made a non-breaking change by adding an option to not include the link, but having the default be link-included?

**Comment by ksstott** [comment]
> Could it be made a non-breaking change by adding an option to not include the link, but having the default be link-included?

Yes that could be an option for a short term solution, in the long term I would feel it makes more sense that the default is that it does not render a link.
We'll consider all the options 😃
